File: testprojects/c#/presentation/gui/orkut/CommunityGUI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.orkut;

namespace CSharpBenchmark.presentation.gui.orkut
{
    //Functional requirement 3.1.2
    //Test case 71: Only class presentation.gui.orkut.CommunityGUI may have a dependency with domain.orkut.Community
    //Result: TRUE
    public class CommunityGUI
    {
        public CommunityGUI()
        {
            //FR5.2
            Console.WriteLine(Community.communityName);
        }
    }
}

File: testresources/java/benchmark_2012/presentation/gui/orkut/CommunityGUI.java
package presentation.gui.orkut;

import domain.orkut.Community;

//Functional requirement 3.1.2
//Test case 71: Only class presentation.gui.orkut.CommunityGUI may have a dependency with domain.orkut.Community
//Result: TRUE
public class CommunityGUI {
	public CommunityGUI(){
		//FR5.2
		System.out.println(Community.communityName);
	}
}